internal/output/markdown.go

```diff
@@ -87,6 +87,10 @@ func calculateMetadataToDisplay(projects []Project) (hasModulePath bool, hasWork
 	// check if any projects that have the same name have different path or workspace
 	for i, p := range sprojects {
 		if i > 0 { // we compare vs the previous item, so skip index 0
+			if p.Diff == nil || len(p.Diff.Resources) == 0 { // ignore the projects that are skipped
+				continue
```

Comment:
I think this doesn't quite cover all cases since when comparing against the 'previous project' (i-1), that could be a project that was supposed to be skipped.   For example, there are two projects with the same name and the first one has no diff, but the second one does, it would still end up as `hasModulePath = true`.

I guess it might be best to filter out the projects without a diff before sorting.